src/rebar3_proper_prv.erl

```diff
@@ -1,4 +1,5 @@
 -module(rebar3_proper_prv).
+-behaviour(provider).
```

Comment:
I didn't put that one here so I could compile and run stuff without depending on rebar3 being in the environment. It was a deliberate omission.